kratos/controllers/register_controllers.cpp

```diff
@@ -0,0 +1,34 @@
+//    |  /           |
```

Comment:
@roigcarlo we should find the time to advance in the cpp registry stuff as this is starting to be populated...

(@ddiezrod please ignore this comment)